Concuerror AFS2018+build.2177.ref65f5340 started at 21 Aug 2018 12:15:41
 Options:
  [{after_timeout,infinity},
   {assertions_only,false},
   {assume_racing,false},
   {depth_bound,500},
   {disable_sleep_sets,false},
   {dpor,optimal},
   {entry_point,{process_info,test_message_queue_len,[]}},
   {exclude_module,[]},
   {files,["/Users/stavros.aronis/git/Concuerror/tests/suites/basic_tests/src/process_info.erl"]},
   {first_process_errors_only,false},
   {ignore_error,[]},
   {instant_delivery,true},
   {interleaving_bound,infinity},
   {keep_going,true},
   {non_racing_system,[]},
   {print_depth,20},
   {quiet,true},
   {scheduling,round_robin},
   {scheduling_bound_type,none},
   {show_races,false},
   {strict_scheduling,false},
   {symbolic_names,true},
   {timeout,5000},
   {treat_as_normal,[]},
   {use_receive_patterns,true}]
################################################################################
Interleaving #1
--------------------------------------------------------------------------------
Errors found:
* Blocked at a 'receive' ("deadlocked"; other processes have exited):
    <P.1> in process_info.erl line 41
     Mailbox contents: [{bar,<P.2>},{ok,<P.3>}]
--------------------------------------------------------------------------------
Event trace:
   1: <P>: <P.1> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-0-'.0>,[]])
    in erlang.erl line 2687
   2: <P>: <P.2> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-1-'.0>,[]])
    in erlang.erl line 2687
   3: <P>: <P.3> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-2-'.0>,[]])
    in erlang.erl line 2687
   4: <P>: <P.4> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-3-'.0>,[]])
    in erlang.erl line 2687
   5: <P>: exits normally
   6: <P.2>: {bar,<P.2>} = <P.1> ! {bar,<P.2>}
    in process_info.erl line 46
   7: <P.2>: {ok,<P.2>} = <P.1> ! {ok,<P.2>}
    in process_info.erl line 47
   8: <P.2>: exits normally
   9: <P.3>: {ok,<P.3>} = <P.1> ! {ok,<P.3>}
    in process_info.erl line 51
  10: <P.3>: exits normally
  11: <P.4>: [{message_queue_len,3}] = erlang:process_info(<P.1>, [message_queue_len])
    in process_info.erl line 57
  12: <P.4>: exits normally
  13: <P.1>: receives message ({ok,<P.2>})
    in process_info.erl line 40
################################################################################
Interleaving #2
--------------------------------------------------------------------------------
Errors found:
* Blocked at a 'receive' ("deadlocked"; other processes have exited):
    <P.1> in process_info.erl line 41
     Mailbox contents: [{bar,<P.2>},{ok,<P.3>}]
--------------------------------------------------------------------------------
Event trace:
   1: <P>: <P.1> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-0-'.0>,[]])
    in erlang.erl line 2687
   2: <P>: <P.2> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-1-'.0>,[]])
    in erlang.erl line 2687
   3: <P>: <P.3> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-2-'.0>,[]])
    in erlang.erl line 2687
   4: <P>: <P.4> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-3-'.0>,[]])
    in erlang.erl line 2687
   5: <P>: exits normally
   6: <P.2>: {bar,<P.2>} = <P.1> ! {bar,<P.2>}
    in process_info.erl line 46
   7: <P.2>: {ok,<P.2>} = <P.1> ! {ok,<P.2>}
    in process_info.erl line 47
   8: <P.2>: exits normally
   9: <P.3>: {ok,<P.3>} = <P.1> ! {ok,<P.3>}
    in process_info.erl line 51
  10: <P.3>: exits normally
  11: <P.1>: receives message ({ok,<P.2>})
    in process_info.erl line 40
  12: <P.4>: [{message_queue_len,2}] = erlang:process_info(<P.1>, [message_queue_len])
    in process_info.erl line 57
  13: <P.4>: exits normally
################################################################################
Interleaving #3
--------------------------------------------------------------------------------
Errors found:
* Blocked at a 'receive' ("deadlocked"; other processes have exited):
    <P.1> in process_info.erl line 41
     Mailbox contents: [{bar,<P.2>},{ok,<P.3>}]
--------------------------------------------------------------------------------
Event trace:
   1: <P>: <P.1> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-0-'.0>,[]])
    in erlang.erl line 2687
   2: <P>: <P.2> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-1-'.0>,[]])
    in erlang.erl line 2687
   3: <P>: <P.3> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-2-'.0>,[]])
    in erlang.erl line 2687
   4: <P>: <P.4> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-3-'.0>,[]])
    in erlang.erl line 2687
   5: <P>: exits normally
   6: <P.2>: {bar,<P.2>} = <P.1> ! {bar,<P.2>}
    in process_info.erl line 46
   7: <P.2>: {ok,<P.2>} = <P.1> ! {ok,<P.2>}
    in process_info.erl line 47
   8: <P.2>: exits normally
   9: <P.4>: [{message_queue_len,2}] = erlang:process_info(<P.1>, [message_queue_len])
    in process_info.erl line 57
  10: <P.4>: exits normally
  11: <P.1>: receives message ({ok,<P.2>})
    in process_info.erl line 40
  12: <P.3>: {ok,<P.3>} = <P.1> ! {ok,<P.3>}
    in process_info.erl line 51
  13: <P.3>: exits normally
################################################################################
Interleaving #4
--------------------------------------------------------------------------------
Errors found:
* Blocked at a 'receive' ("deadlocked"; other processes have exited):
    <P.1> in process_info.erl line 41
     Mailbox contents: [{bar,<P.2>},{ok,<P.3>}]
--------------------------------------------------------------------------------
Event trace:
   1: <P>: <P.1> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-0-'.0>,[]])
    in erlang.erl line 2687
   2: <P>: <P.2> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-1-'.0>,[]])
    in erlang.erl line 2687
   3: <P>: <P.3> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-2-'.0>,[]])
    in erlang.erl line 2687
   4: <P>: <P.4> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-3-'.0>,[]])
    in erlang.erl line 2687
   5: <P>: exits normally
   6: <P.2>: {bar,<P.2>} = <P.1> ! {bar,<P.2>}
    in process_info.erl line 46
   7: <P.2>: {ok,<P.2>} = <P.1> ! {ok,<P.2>}
    in process_info.erl line 47
   8: <P.2>: exits normally
   9: <P.1>: receives message ({ok,<P.2>})
    in process_info.erl line 40
  10: <P.4>: [{message_queue_len,1}] = erlang:process_info(<P.1>, [message_queue_len])
    in process_info.erl line 57
  11: <P.4>: exits normally
  12: <P.3>: {ok,<P.3>} = <P.1> ! {ok,<P.3>}
    in process_info.erl line 51
  13: <P.3>: exits normally
################################################################################
Interleaving #5
--------------------------------------------------------------------------------
Errors found:
* Blocked at a 'receive' ("deadlocked"; other processes have exited):
    <P.1> in process_info.erl line 41
     Mailbox contents: [{bar,<P.2>},{ok,<P.2>}]
--------------------------------------------------------------------------------
Event trace:
   1: <P>: <P.1> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-0-'.0>,[]])
    in erlang.erl line 2687
   2: <P>: <P.2> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-1-'.0>,[]])
    in erlang.erl line 2687
   3: <P>: <P.3> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-2-'.0>,[]])
    in erlang.erl line 2687
   4: <P>: <P.4> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-3-'.0>,[]])
    in erlang.erl line 2687
   5: <P>: exits normally
   6: <P.2>: {bar,<P.2>} = <P.1> ! {bar,<P.2>}
    in process_info.erl line 46
   7: <P.3>: {ok,<P.3>} = <P.1> ! {ok,<P.3>}
    in process_info.erl line 51
   8: <P.2>: {ok,<P.2>} = <P.1> ! {ok,<P.2>}
    in process_info.erl line 47
   9: <P.2>: exits normally
  10: <P.3>: exits normally
  11: <P.4>: [{message_queue_len,3}] = erlang:process_info(<P.1>, [message_queue_len])
    in process_info.erl line 57
  12: <P.4>: exits normally
  13: <P.1>: receives message ({ok,<P.3>})
    in process_info.erl line 40
################################################################################
Interleaving #6
--------------------------------------------------------------------------------
Errors found:
* Blocked at a 'receive' ("deadlocked"; other processes have exited):
    <P.1> in process_info.erl line 41
     Mailbox contents: [{bar,<P.2>},{ok,<P.2>}]
--------------------------------------------------------------------------------
Event trace:
   1: <P>: <P.1> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-0-'.0>,[]])
    in erlang.erl line 2687
   2: <P>: <P.2> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-1-'.0>,[]])
    in erlang.erl line 2687
   3: <P>: <P.3> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-2-'.0>,[]])
    in erlang.erl line 2687
   4: <P>: <P.4> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-3-'.0>,[]])
    in erlang.erl line 2687
   5: <P>: exits normally
   6: <P.2>: {bar,<P.2>} = <P.1> ! {bar,<P.2>}
    in process_info.erl line 46
   7: <P.3>: {ok,<P.3>} = <P.1> ! {ok,<P.3>}
    in process_info.erl line 51
   8: <P.2>: {ok,<P.2>} = <P.1> ! {ok,<P.2>}
    in process_info.erl line 47
   9: <P.2>: exits normally
  10: <P.3>: exits normally
  11: <P.1>: receives message ({ok,<P.3>})
    in process_info.erl line 40
  12: <P.4>: [{message_queue_len,2}] = erlang:process_info(<P.1>, [message_queue_len])
    in process_info.erl line 57
  13: <P.4>: exits normally
################################################################################
Interleaving #7
--------------------------------------------------------------------------------
Errors found:
* Blocked at a 'receive' ("deadlocked"; other processes have exited):
    <P.1> in process_info.erl line 41
     Mailbox contents: [{bar,<P.2>},{ok,<P.2>}]
--------------------------------------------------------------------------------
Event trace:
   1: <P>: <P.1> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-0-'.0>,[]])
    in erlang.erl line 2687
   2: <P>: <P.2> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-1-'.0>,[]])
    in erlang.erl line 2687
   3: <P>: <P.3> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-2-'.0>,[]])
    in erlang.erl line 2687
   4: <P>: <P.4> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-3-'.0>,[]])
    in erlang.erl line 2687
   5: <P>: exits normally
   6: <P.2>: {bar,<P.2>} = <P.1> ! {bar,<P.2>}
    in process_info.erl line 46
   7: <P.3>: {ok,<P.3>} = <P.1> ! {ok,<P.3>}
    in process_info.erl line 51
   8: <P.3>: exits normally
   9: <P.4>: [{message_queue_len,2}] = erlang:process_info(<P.1>, [message_queue_len])
    in process_info.erl line 57
  10: <P.4>: exits normally
  11: <P.1>: receives message ({ok,<P.3>})
    in process_info.erl line 40
  12: <P.2>: {ok,<P.2>} = <P.1> ! {ok,<P.2>}
    in process_info.erl line 47
  13: <P.2>: exits normally
################################################################################
Interleaving #8
--------------------------------------------------------------------------------
Errors found:
* Blocked at a 'receive' ("deadlocked"; other processes have exited):
    <P.1> in process_info.erl line 41
     Mailbox contents: [{bar,<P.2>},{ok,<P.2>}]
--------------------------------------------------------------------------------
Event trace:
   1: <P>: <P.1> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-0-'.0>,[]])
    in erlang.erl line 2687
   2: <P>: <P.2> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-1-'.0>,[]])
    in erlang.erl line 2687
   3: <P>: <P.3> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-2-'.0>,[]])
    in erlang.erl line 2687
   4: <P>: <P.4> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-3-'.0>,[]])
    in erlang.erl line 2687
   5: <P>: exits normally
   6: <P.2>: {bar,<P.2>} = <P.1> ! {bar,<P.2>}
    in process_info.erl line 46
   7: <P.3>: {ok,<P.3>} = <P.1> ! {ok,<P.3>}
    in process_info.erl line 51
   8: <P.3>: exits normally
   9: <P.1>: receives message ({ok,<P.3>})
    in process_info.erl line 40
  10: <P.4>: [{message_queue_len,1}] = erlang:process_info(<P.1>, [message_queue_len])
    in process_info.erl line 57
  11: <P.4>: exits normally
  12: <P.2>: {ok,<P.2>} = <P.1> ! {ok,<P.2>}
    in process_info.erl line 47
  13: <P.2>: exits normally
################################################################################
Interleaving #9
--------------------------------------------------------------------------------
Errors found:
* Blocked at a 'receive' ("deadlocked"; other processes have exited):
    <P.1> in process_info.erl line 41
     Mailbox contents: [{bar,<P.2>},{ok,<P.3>}]
--------------------------------------------------------------------------------
Event trace:
   1: <P>: <P.1> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-0-'.0>,[]])
    in erlang.erl line 2687
   2: <P>: <P.2> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-1-'.0>,[]])
    in erlang.erl line 2687
   3: <P>: <P.3> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-2-'.0>,[]])
    in erlang.erl line 2687
   4: <P>: <P.4> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-3-'.0>,[]])
    in erlang.erl line 2687
   5: <P>: exits normally
   6: <P.2>: {bar,<P.2>} = <P.1> ! {bar,<P.2>}
    in process_info.erl line 46
   7: <P.4>: [{message_queue_len,1}] = erlang:process_info(<P.1>, [message_queue_len])
    in process_info.erl line 57
   8: <P.4>: exits normally
   9: <P.2>: {ok,<P.2>} = <P.1> ! {ok,<P.2>}
    in process_info.erl line 47
  10: <P.2>: exits normally
  11: <P.3>: {ok,<P.3>} = <P.1> ! {ok,<P.3>}
    in process_info.erl line 51
  12: <P.3>: exits normally
  13: <P.1>: receives message ({ok,<P.2>})
    in process_info.erl line 40
################################################################################
Interleaving #10
--------------------------------------------------------------------------------
Errors found:
* Blocked at a 'receive' ("deadlocked"; other processes have exited):
    <P.1> in process_info.erl line 41
     Mailbox contents: [{bar,<P.2>},{ok,<P.2>}]
--------------------------------------------------------------------------------
Event trace:
   1: <P>: <P.1> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-0-'.0>,[]])
    in erlang.erl line 2687
   2: <P>: <P.2> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-1-'.0>,[]])
    in erlang.erl line 2687
   3: <P>: <P.3> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-2-'.0>,[]])
    in erlang.erl line 2687
   4: <P>: <P.4> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-3-'.0>,[]])
    in erlang.erl line 2687
   5: <P>: exits normally
   6: <P.2>: {bar,<P.2>} = <P.1> ! {bar,<P.2>}
    in process_info.erl line 46
   7: <P.4>: [{message_queue_len,1}] = erlang:process_info(<P.1>, [message_queue_len])
    in process_info.erl line 57
   8: <P.4>: exits normally
   9: <P.3>: {ok,<P.3>} = <P.1> ! {ok,<P.3>}
    in process_info.erl line 51
  10: <P.2>: {ok,<P.2>} = <P.1> ! {ok,<P.2>}
    in process_info.erl line 47
  11: <P.2>: exits normally
  12: <P.3>: exits normally
  13: <P.1>: receives message ({ok,<P.3>})
    in process_info.erl line 40
################################################################################
Interleaving #11
--------------------------------------------------------------------------------
Errors found:
* Blocked at a 'receive' ("deadlocked"; other processes have exited):
    <P.1> in process_info.erl line 41
     Mailbox contents: [{bar,<P.2>},{ok,<P.2>}]
--------------------------------------------------------------------------------
Event trace:
   1: <P>: <P.1> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-0-'.0>,[]])
    in erlang.erl line 2687
   2: <P>: <P.2> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-1-'.0>,[]])
    in erlang.erl line 2687
   3: <P>: <P.3> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-2-'.0>,[]])
    in erlang.erl line 2687
   4: <P>: <P.4> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-3-'.0>,[]])
    in erlang.erl line 2687
   5: <P>: exits normally
   6: <P.3>: {ok,<P.3>} = <P.1> ! {ok,<P.3>}
    in process_info.erl line 51
   7: <P.3>: exits normally
   8: <P.4>: [{message_queue_len,1}] = erlang:process_info(<P.1>, [message_queue_len])
    in process_info.erl line 57
   9: <P.4>: exits normally
  10: <P.1>: receives message ({ok,<P.3>})
    in process_info.erl line 40
  11: <P.2>: {bar,<P.2>} = <P.1> ! {bar,<P.2>}
    in process_info.erl line 46
  12: <P.2>: {ok,<P.2>} = <P.1> ! {ok,<P.2>}
    in process_info.erl line 47
  13: <P.2>: exits normally
################################################################################
Interleaving #12
--------------------------------------------------------------------------------
Errors found:
* Blocked at a 'receive' ("deadlocked"; other processes have exited):
    <P.1> in process_info.erl line 41
     Mailbox contents: [{bar,<P.2>},{ok,<P.2>}]
--------------------------------------------------------------------------------
Event trace:
   1: <P>: <P.1> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-0-'.0>,[]])
    in erlang.erl line 2687
   2: <P>: <P.2> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-1-'.0>,[]])
    in erlang.erl line 2687
   3: <P>: <P.3> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-2-'.0>,[]])
    in erlang.erl line 2687
   4: <P>: <P.4> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-3-'.0>,[]])
    in erlang.erl line 2687
   5: <P>: exits normally
   6: <P.3>: {ok,<P.3>} = <P.1> ! {ok,<P.3>}
    in process_info.erl line 51
   7: <P.3>: exits normally
   8: <P.1>: receives message ({ok,<P.3>})
    in process_info.erl line 40
   9: <P.4>: [{message_queue_len,0}] = erlang:process_info(<P.1>, [message_queue_len])
    in process_info.erl line 57
  10: <P.4>: exits normally
  11: <P.2>: {bar,<P.2>} = <P.1> ! {bar,<P.2>}
    in process_info.erl line 46
  12: <P.2>: {ok,<P.2>} = <P.1> ! {ok,<P.2>}
    in process_info.erl line 47
  13: <P.2>: exits normally
################################################################################
Interleaving #13
--------------------------------------------------------------------------------
Errors found:
* Blocked at a 'receive' ("deadlocked"; other processes have exited):
    <P.1> in process_info.erl line 41
     Mailbox contents: [{bar,<P.2>},{ok,<P.3>}]
--------------------------------------------------------------------------------
Event trace:
   1: <P>: <P.1> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-0-'.0>,[]])
    in erlang.erl line 2687
   2: <P>: <P.2> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-1-'.0>,[]])
    in erlang.erl line 2687
   3: <P>: <P.3> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-2-'.0>,[]])
    in erlang.erl line 2687
   4: <P>: <P.4> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-3-'.0>,[]])
    in erlang.erl line 2687
   5: <P>: exits normally
   6: <P.4>: [{message_queue_len,0}] = erlang:process_info(<P.1>, [message_queue_len])
    in process_info.erl line 57
   7: <P.4>: exits normally
   8: <P.2>: {bar,<P.2>} = <P.1> ! {bar,<P.2>}
    in process_info.erl line 46
   9: <P.2>: {ok,<P.2>} = <P.1> ! {ok,<P.2>}
    in process_info.erl line 47
  10: <P.2>: exits normally
  11: <P.3>: {ok,<P.3>} = <P.1> ! {ok,<P.3>}
    in process_info.erl line 51
  12: <P.3>: exits normally
  13: <P.1>: receives message ({ok,<P.2>})
    in process_info.erl line 40
################################################################################
Interleaving #14
--------------------------------------------------------------------------------
Errors found:
* Blocked at a 'receive' ("deadlocked"; other processes have exited):
    <P.1> in process_info.erl line 41
     Mailbox contents: [{bar,<P.2>},{ok,<P.2>}]
--------------------------------------------------------------------------------
Event trace:
   1: <P>: <P.1> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-0-'.0>,[]])
    in erlang.erl line 2687
   2: <P>: <P.2> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-1-'.0>,[]])
    in erlang.erl line 2687
   3: <P>: <P.3> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-2-'.0>,[]])
    in erlang.erl line 2687
   4: <P>: <P.4> = erlang:spawn(erlang, apply, [#Fun<process_info.'-test_with_messages/1-fun-3-'.0>,[]])
    in erlang.erl line 2687
   5: <P>: exits normally
   6: <P.4>: [{message_queue_len,0}] = erlang:process_info(<P.1>, [message_queue_len])
    in process_info.erl line 57
   7: <P.4>: exits normally
   8: <P.2>: {bar,<P.2>} = <P.1> ! {bar,<P.2>}
    in process_info.erl line 46
   9: <P.3>: {ok,<P.3>} = <P.1> ! {ok,<P.3>}
    in process_info.erl line 51
  10: <P.2>: {ok,<P.2>} = <P.1> ! {ok,<P.2>}
    in process_info.erl line 47
  11: <P.2>: exits normally
  12: <P.3>: exits normally
  13: <P.1>: receives message ({ok,<P.3>})
    in process_info.erl line 40
################################################################################
Exploration completed!
################################################################################
Tips:
--------------------------------------------------------------------------------
* Check `--help attributes' for info on how to pass options via module attributes.
* Running without a scheduling_bound corresponds to verification and may take a long time.
* Increase '--print_depth' if output/graph contains "...".
* Each of the first 10 interleavings explored so far had some error. This can make later debugging difficult, as the generated report will include too much info. Consider refactoring your code, or using the appropriate options to filter out irrelevant errors.

################################################################################
Info:
--------------------------------------------------------------------------------
* Writing results in /Users/stavros.aronis/git/Concuerror/tests/results/basic_tests/results/process_info-test_message_queue_len-inf-dpor.txt
* Automatically instrumented module io_lib
* Showing PIDs as "<symbolic name(/last registered name)>" ('-h symbolic_names').
* Instrumented & loaded module process_info
* Automatically instrumented module erlang
* Continuing after error (-k)
* You can see pairs of racing instructions (in the report and '--graph') with '--show_races true'

################################################################################
Done at 21 Aug 2018 12:15:41 (Exit status: error)
  Summary: 14 errors, 14/14 interleavings explored
